doc/tutorials/asset.ipynb

```diff
@@ -0,0 +1,933 @@
A6XgWCELBaclu5Jz6sZ9IEIoABOjMAUD8aEcRCy+0uXbTmK59cfp7OaeBvQhCWSAIBad5oaMgWTeWVsOo3dSBgHtQBgAgftShiIUXyovbg1BOBokCgYDtPjTgdwShLBCEgtO80FGIxu0dCLgHZQAA4kcdilh4YZSN3TId7764OQ+izS7gnEYmYmFywCdS1fCyoCQAAIC7uCXYkgzx7ku0Pqqf8gbwM0ZCMRIKDnLi6qbbrpi6LT1IPcoAAMSPOhR+43SZdvNoIzenDUg1RkIBacBoImQCyjkAAAiiXwCgPAShAAdlwrBgOhrIhHIOAE6hHYXfuLlfkI71pDjHgXBMx2M6HjwmGUOc3bygIwAAABBNvFPeUnHnOvrYwD7cHc8CQSg4xckGKBlBqExcf4JOgTPI19QivwEg83i17reb7lj3L95gEus1AalFEMoCQSg4xckgTywNdbTXZmIQKhP3ORXI19QivwEg83i17reb7nj2r6L3WPWBo02L80JeAl5EEMoCQSg4JdWdhViDTV7tzCQiE/c5FcjX1CK/ASDzeLXudzIIVdFF2Wijnqx4IS8BLyIIZYEgFJyS6s5CrMGmWNLn1SHgZXm1A+d25Gtqkd8AkHm8Wvc7GYSK57ut5OTk2OrX+qU/DKQSQSgLBKHgFD8Fobza8SnLL/vhNuRrapHfAJB5vFr3eyEIZfd7vHoMgHSKFm+pkoa0IA2I3qM0bhULAAAAtwr+dkn27xX6wED6EYTKEFYVLkEo57i9gcvEY+/2Y+JV5Gtqkd8AkHm8WvfbTXfwdaVfH/x/LH1Wq4vuZWViHxhwG6bjZch0PIaQekc8o9aiHd9kjICj7AAAAGSOdM2gSLTPaWcKXrx9WPrDQOxYE8oCQSgqTjdy4g4hyU4PI+sAAAD8KV2/G5wIQpXts8bbX6U/DMSOIJQFglAEodzIbcfKKsBl1ehSngAAiB3rdsJt/BSEKvv+eM83+sNA7AhCWcikIBQdHO9w20goK24LlAEA4FW0qXCbdJXJRPuz5Y1WCn5OMveNcxcoH0EoC5kUhIJ3xNOgpboRpNEFACA5aFPhNl4tk+WtCRVMP0EoIHWixVu4Ox6Actm50wgAAACQTlZ32UsW+sNA8jASipFQcJnyhiJHe87JKzF27jSSzO8DACCTMJoCbpOKZR5SfVOdREdC0R8GYsd0PAsEoZAqyWpoozWcqQ5CcTcQBLHeHAAkJpE2nDo4M3j1OJeX7lT3XZ0IQtEfhlfPzVQhCGWBIBRSJVkNbbQG0Mm7c3CFNv3c3MBRPgAgMYnU8dTBmcGrx9mJYJCd86W8z473fPPqMYCzKBflIwhlgSAUUiVZd7yLNsfdySsxVK7p5+Zj4Oa0AYDfUQdnBq8e51SNSCr7Picu3nn1GMBZlIvysTA5kELJWLwwlkUVnRwV48TijvC3QCDgmpFaQGluHlUIAKWxEHb8nKjXvd4fpv2DmzASipFQcICdxQsripLbXQDRzmfB29x8lSVaOXVL+oDS3HwuAfGgTPtXRf1ALxzndI2EQiTyzRnka/kYCQX4QLT1n+LFVREASJzb6lK3pQcAykrnyCLqSCC9GAnFSCg4wM4dNCpq7FKxCDnRe29w83EKBAKOLowPJJNT55LbzlG3pQfO4ce0f1XUl/TCcXaiLkrGZ2ZiHZmJ+5wK1MHlY2FyCwSh4BSnGkgrOTk5cVd2NEje4PYGjnIEryAIBcAr/HAeO7EPyegTxZIut/fB7PJDeYL3EISyQBAKTklGRW/37niJNCA0SEgGyhG8IpODUIlcsACQem6rV+Lh1n2IJV1u3YdY+WU/4C2sCQWkUKLz3KNddfH6nTngT5RLeEUml1WCUIC3+GHNpEyuc92GYwE3YSQUI6HgQtGuViT7KoZfhhgDQDq5rS5lrTYAiUjnqJlU1KexfAcjiID4MR3PAkEouFWqglAAAH+ivQAQr3TWH26ru9yWHsBLmI4HeJzVVRsAAADA6+jnApmDIBSQQokMMWYuNwAAAEpz23Rgu8qm2639XLemC/AypuMxHQ8OstPAWg3ptRr6G+0zvNDRAACklld/mAKIjRPTxZy6y3PpOqi8vm6i3w3AHZiOB6RBvFdPgu/j6gsAIB4EnADEKxn9T6vPSGa9RKAd8C5GQjESCg6yc5WnvKs7dt9PQwwAAOAMN/ez3LpwdkXpstPHDQayrPI62ue7+VgBmYa741kgCAUnlG787FxJKq+jYHc6n1s7IPAvOnn7kBcA4O+6MJ39rHimtbmhD2iVrpycnFDaoz1fWnnlhztJI1F+rrPcgiCUBYJQcEIsc9yl2BpGGly4BWVuH/ICAPxdF6Zz3+IZUeSGfI/WHw6mLdF00ydGoigrzmNNKMAlWOcJAABncYUbmcKr/Uon0m113sMbqLMzCyOhGAmFJKtoeHEiFWq8V32o2JFsXD3ah7yIH3UTnMJ5mXp+znM3j4Ryq0AgYPuu0PGIZeaB04ukI3HpKOdePbe8hOl4FghCISiZP4QSrdDKS0u057w6VBveRZnax295kcrAkN/yDu5B2Uo9P+d5pgShkl3/O5l2q7TGMrrKL2XTLwhC+RNBKAsEoRCUzErIqTnuEkEouAdlah+/5UUq98dveQf3oGylnp/zPJ2jNr1cJ6c6gGYVhMrJyXF0RBaSgyCUP7EmFJAi0a7CJKMDw3BiuIVX16BwAnkBAP6uC9PZ1yovX90+pbmiMpHM9Ef7rkGDBvm6bCJ+lIv0YSQUI6Gg1ETC7X5Hea+L9pxXb98LwH28fNUdCHL7j3MgGbw8cinZ3xdtTVY7MwaQftTZ/sR0PAsEoRDk9SBUsr4bAAhCAYA3EISy91m0NUB6MB0PyGAMNwXgRtRNAIBYWY2aKQ9tDeAuBKEAuatxijUtpRviaMNWGc4KwK5U1ofUTch0TEGBm7ipP1weu+kMBAIaNGgQ5xTgMkzHYzoeUiQZQ4GtPiORzwuiE5x85Ole5APgHM4v72OaEBL
```

Comment:
```suggestion
    "#### With the sequences of synchronous events `sses` we found a permutation of the spiketrains that reveals the synfire chain as in the ground truth ordering in the last Figure, shown below."
```

missing article

Reply:
sorry, from here on the web interface for reviews mixed up / shifted the line numbers somehow